src/constraint_system/arithmetic.rs

```diff
@@ -303,6 +367,54 @@ impl<E: PairingEngine, P: TEModelParameters<BaseField = E::Fr>>
 
         self.big_mul_gate(a, b, c, Some(d), q_m, q_o, q_c, q_4, pi)
     }
+
+    /// Adds a [`StandardComposer::big_arith_gate`] with the left, right
+    /// , fourth inputs and corresponding coefficients, computing & returning
+    /// the output (result) [`Variable`] and adding the corresponding arith
+    /// constraint.
+    ///
+    /// This type of gate is usually used when we don't need to have
+    /// the largest amount of performance and the minimum circuit-size
+    /// possible, since it defaults set `q_o` to `-1` to reduce the verbosity.
+    ///
+    /// Equation: `(a*b)*q_m + a*q_l + b*q_r + d*q_4 + q_c + PI = c`
+    /// ### Returns
+    /// `c`
+    pub fn big_arith(
+        &mut self,
+        q_m: E::Fr,
+        a: Variable,
+        b: Variable,
+        q_l: E::Fr,
+        q_r: E::Fr,
+        q_4_d: Option<(E::Fr, Variable)>,
+        q_c: E::Fr,
+        pi: Option<E::Fr>,
+    ) -> Variable {
+        let q_o = -E::Fr::one();
```

Comment:
This assigment is not needed. You can directly add it in the `big_arith_gate` call and if you want, highlight with a comment that corresponds to `q_o`.

Clippy will probably complain about it.

Reply:
fixed